test/data/gvcf_test_cases/format_fields_integrated.yml

```diff
@@ -119,7 +119,7 @@ truth_output_vcf:
       ##contig=<ID=21,length=48129895>
       #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12878	NA12879
       21	10009461	.	T	A	.	.	.	GT:AD:DP:GQ:SB:RNC	0/1:10,2:12:16:1,9,1,1:..	0/1:10,2:12:16:1,9,1,1:..
-      21	10009462	.	T	A	0	.	.	GT:DP:GQ:AD:RNC	0/0:14:36:14,0:..	0/1:12:16:10,2:..
-      21	10009463	.	C	G	0	.	.	GT:DP:GQ:AD:RNC	0/0:14:36:14,0:..	0/1:12:16:10,2:..
+      21	10009462	.	T	A	0	.	.	GT:DP:GQ:AD:SB:RNC	0/0:14:36:14,0:.,.,.,.:..	0/1:12:16:10,2:1,9,1,1:..
```

Comment:
[Follow up] it seems like `htslib` doesn't automatically drop the trailing field when it's missing. This is the output from `format_fields_default_missing.yml` when `RNC` is moved forward so that `SB` and `SBF`, both of which are missing, are now the *trailing* fields:

```
#CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO    FORMAT  A       B
A       1000    .       C       CG      0       .       .       GT:RNC:DP:AD:SB:SBF     0/0:..:2:1,0:.,.,.,.:.,.,.,.    1/1:..:.:1,19:1,2,3,4:1.1,2.2,3.3,4.4
```